developer/developer-faqs.md

```diff
@@ -48,6 +48,20 @@ Since 2014 [Google has indexed JavaScript when crawling websites](https://webmas
 
 3. [meteor-dochead](https://github.com/kadirahq/meteor-dochead): Automatically add `<meta>` tags are for products using the [meteor-dochead](https://github.com/kadirahq/meteor-dochead) package which uses the title, description and `details` fields of the product to render SEO-friendly data. You may need to use a tool like [SEO Inspector](https://chrome.google.com/webstore/detail/seo-inspector/iejckekdjogeeilmllnabmgkbbmedeal?hl=en) to see this data.
 
+## Extending Reaction
+
+### How can I add static pages/Integrate with a CMS?
```

Comment:
How can I code a static page? What about handling multiple pages and images with a content management system (CMS)?

Reply:
I would separate out `What about handling multiple pages and images with a content management system (CMS)?` into a 2nd question.